src/components/container/DocumentLibraryPreview/DocumentLibraryPreview.tsx

```
@@ -0,0 +1,90 @@
+
+    return (
+        <div ref={props.innerRef} className="mdhui-document-library-preview">
+            {!surveyFiles && <LoadingIndicator />}
+            {surveyFiles &&
+                <Action title={` ${language("uploaded-documents")}`}
+                    className="font-size"
+                    indicatorPosition="default"
+                    subtitle={surveyFiles.length === 0
+                        ? language("upload-documents-subtitle")
+                        : ""}
+                    children={surveyFiles.length > 0
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)